instance/tests/models/test_openedx_instance.py

```diff
@@ -847,3 +848,38 @@ def test_shut_down_reminder(self):
         msg = r"Use archive\(\) to shut down all of an instances app servers and remove it from the instance list."
         with self.assertRaisesRegex(AttributeError, msg):
             instance.shut_down()
+
+    def _verify_vm_dns_records(self, domain, active_vm_ips):
+        """
+        Helper function to verify a set of DNS records for active VMs.
+        """
+        dns_ips = set()
+        vm_indexes = set()
```

Comment:
@itsjeyd Both indexes and indices are standard plural forms, but for different meanings of the word.  In this case, it's definitely indices, but for e.g. a database index it's indexes.